plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -155,6 +156,9 @@ private void appendColumn(PartialRow row, Page page, int position, int channel,
         else if (TIMESTAMP_MILLIS.equals(type)) {
             row.addLong(destChannel, truncateEpochMicrosToMillis(TIMESTAMP_MILLIS.getLong(block, position)));
         }
+        else if (DATE.equals(type)) {
```

Comment:
We usually put DATE type before TIMESTAMP types.

Reply:
It's just a convention. You can confirm the entire list in `PostgreSqlClient#toColumnMapping`.

Reply:
I think it is more on the size of the data for temporal types - like date, timestamp and timestamp with tz

Reply:
Thanks @ebyhr  and @Praveen2112